CHANGELOG.md

```diff
@@ -4,7 +4,21 @@ This file summarizes notable changes introduced in aktualizr version. It roughly
 
 Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new release is issued. Thus `N` does not necessarily map to months of the year.
 
-## [??? (unreleased)]
+## [upcoming release]
+
+### Special considerations
+
+As a result of changes to the IP/POSIX Secondary protocol (see below), users of these Secondaries will need to take special care when upgrading their devices. The new version of aktualizr (e.g. the Primary) is backwards compatible and will work with both old and new versions of the protocol. However, the aktualizr-secondary is *not*. This means that if you are upgrading a device with IP/POSIX Secondaries, you should update the Primary ECU first, and if that is successful, then update your Secondaries.
+
+### Added
+
+- SetInstallationRawReport API call to enable setting a custom raw report field in the device installation result: [PR](https://github.com/advancedtelematic/aktualizr/pull/1628)
+- Allow ECU re-registration to support adding, removing, and replacing Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1686)
```

Comment:
I don't think I agree with this one. I want to stress that this feature is now allowed going forward. We generally use the present tense for anything that is added like this. The past tense is only used for actions that were taken once, usually referring specifically to the code itself. I see that we aren't consistent on that, but present tense is much clearer here IMO.

Reply:
I see your point. Can we maybe rephrase it?
```
- ECU re-registration now supports adding, removing, and replacing Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1686)
```

Reply:
It's tricky. Your suggestion isn't completely accurate, because ECU re-registration wasn't possible before. I'm a bit stuck on how to improve this while remaining technically precise.

Reply:
Aha, I misunderstood it, sorry.
What about this one?
```
Now, you can re-register ECUs to support adding, removing, and replacing Secondaries: [PR](https://github.com/advancedtelematic/aktualizr/pull/1686)
```

Reply:
Yep, that works. I took the idea and expanded on it just a bit.